moveit_core/robot_state/test/robot_state_test.cpp

```diff
@@ -714,6 +738,267 @@ TEST_F(OneRobot, rigidlyConnectedParent)
   EXPECT_EQ(nullptr, state.getRigidlyConnectedParentLinkModel("/"));
 }
 
+TEST(getJacobian, RevoluteJoints)
+{
+  constexpr char kRobotURDF[] = R"(
```

Comment:
nit, instead of raw xml, simple robots are sometimes easier to build and validate using the [RobotModelBuilder](https://github.com/ros-planning/moveit2/blob/main/moveit_core/utils/include/moveit/utils/robot_model_test_utils.h#L96) (no need to change this here, though)

Reply:
Thanks for the pointer! not sure it would make it more readable though, but good to know this exists!